### ISSUE TITLE
Default colors produce some invisible text in editor in high contrast white

### ISSUE THREAD

**Issue by huguesv** [Microsoft/PTVS#2636]
`**`, `.`, `(`, `)`, `+`, `=` are invisible in that theme.

As you can see from the screenshot, C# doesn't have this issue.

![image](https://cloud.githubusercontent.com/assets/1696845/26744125/7dcc63ce-4799-11e7-84e5-9ed3655c8ff2.png)

It's fixable by changing the Python colors in Fonts and Colors in tools/options.  All Python items are set to Default for both foreground and background, and Default for both is White. Most non-Python items are not like this.

![image](https://cloud.githubusercontent.com/assets/1696845/26744454/d4342d18-479a-11e7-833a-f0723abd019f.png)
